src/app/components/SelectField.tsx

```diff
@@ -2,11 +2,11 @@ import * as React from 'react'
 import classnames from 'classnames'
 
 type ListProps = {
-  list: any[]
+  list: { name: string | number }[]
   placeholder: string
   setValid?: React.Dispatch<React.SetStateAction<boolean>>
-  value: any
-  setValue?: any
+  value: string | undefined
+  setValue?: React.Dispatch<React.SetStateAction<string | undefined>>
```

Comment:
Why `value` have type `string | undefined` and not just `string`? Please explain

Reply:
This value (month, day, year) is taken from the signUpPage state in Redux. And signUpPage is optional because they are injected when the components are mounted sometime in application's life.